src/main/java/com/vestmark/bitbucket/plugin/MergeConflictDetector.java

```diff
@@ -128,6 +130,19 @@ public void addResult(Branch toBranch, List<GitMergeConflict> mergeConflicts,
     mergeResults.add(new MergeResult(toBranch, mergeConflicts, messages, files));
   }
 
+  public boolean isRelated(Branch toBranch)
+  {
+    String family = toBranchName.substring(toBranchName.lastIndexOf("/")).replaceAll("\\/([^\\.|-]*).*", "$1");
```

Comment:
Is this how the bitbucket 'family' logic works?
How I'm reading this ->
`release/foo/bar-20 is in the same family as release/bar/bar-21`
does that seem right?
Would making use of the release prefix make this easier?
```public boolean isRelated(Branch otherBranch, String releasePrefix)
{
//assumes this branch has been prescreened as a release
  String familyName = toBranchName.substring(releasePrefix.length() + 1).replaceAll("\\/([^\\.|-]*).*", "$1");
  String otherFamilyName = toBranch.getDisplayId().substring(releasePrefix.length() +1).replaceAll("\\/([^\\.|-]*).*", "$1");
...
```
In my example above, the families would then be interpreted at the first level, rather than the last
```release/foo/bar-20 != release/bar/bar-21
release/foo/bar-20 == release/foo/foo2```